eth2/types/src/beacon_state_tests.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{
```

Comment:
What do you think about putting the tests within their own directory, and importing from the local file. So we don't need to load from lib.rs
More specifically, moving beacon_state_tests.rs to ./beacon_state/tests.rs and adding `mod tests` to `beacon_state.rs`

Reply:
Sounds good. Do you think `beacon_state.rs` should live in `beacon_state/` or just the tests?

Reply:
Pierre does it with just the tests, so would be leaning towards that approach.

Reply:
Cool, works for me. I'll do it now.

Reply:
Done!